strap.sh

```diff
@@ -68,7 +68,7 @@ verify_keyring()
   if ! gpg --keyserver pgp.mit.edu \
      --recv-keys 4345771566D76038C7FEB43863EC0ADBEA87E4E3 > /dev/null 2>&1
   then
-    if ! gpg --keyserver hkp://pool.sks-keyservers.net \
```

Comment:
is this one dead or something?

Reply:
We had a lot of people telling us, that the solution for the key problem in our FAQ does not work. I faced the same issue on occasions. This solves the problem for me and at least two other people. 
So instead of pointing people to our FAQ/IRC/discord I changed it to something, that seems to work for people with problems with the MIT servers.

Reply:
Ok make sense